src/app/content/highlights/constants.ts

```diff
@@ -14,5 +14,6 @@ export const highlightIndicatorSizeForBlock = 1.2;
 export const highlightIndicatorSize = 0.9;
 
 export const highlightKeyCombination: KeyCombinationOptions = {
-  key: 'h',
+  key: '˙', // key isn't h when alt is pressed
```

Comment:
Is this consistent for different keyboards? Otherwise we may have to use the `code` property instead or something.

Reply:
It's standard according to [this](https://www.toptal.com/developers/keycode/table).

Reply:
In my Windows tests, this is still interpreted as `h`:

```
{
 "key": "h",
 "keyCode": 72,
 "which": 72,
 "code": "KeyH",
 "location": 0,
 "altKey": true,
 "ctrlKey": false,
 "metaKey": false,
 "shiftKey": false,
 "repeat": false
}
```

I think we'll need to check the modifiers in addition to the key/code.

Reply:
I was checking the altKey modifier anyway. I just have to use `code` instead of `key`.